### ISSUE TITLE
[BITV] 9.1.4.1/3.1 - Styling menubar - For many elements like "B" and "I", selected styles are visually marked by a pale blue background (1.5:1 to white) and a change of text color (black to blue, 2.2:1). Both remain below the contrast difference of 3:1 that is needed to indicate the state visually. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36946]
- [ ] Fix
- [ ] Backport

<img width="237" alt="7d269fef5d1ea6ddaaa3bbb3e5e4230b" src="https://user-images.githubusercontent.com/1315170/222219980-b1ea87af-e3be-4fb5-b9a8-3b590d5aea06.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-45ae33ca9f-v3-n1

</details>

- [ ] 9.1.4.1/10.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-45ae33ca9f-v10-n1
